x-pack/plugins/search_playground/public/embedable.tsx

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
nitpick: file should be named `embeddable.tsx` not `embedable.tsx`

Reply:
This file is using jsx for creating lazyRender function